settings.gradle

```
@@ -7,6 +7,7 @@ include ':drivers:sqlite-driver'
 include ':drivers:sqljs-driver'
 include ':drivers:driver-test'
 include ':extensions:android-paging'
+include ':extensions:android-paging3-coroutines'
```

```suggestion
include ':extensions:android-paging3'
```
Need to rename folder as well for this

---

Done!